test/util.jl

```diff
@@ -95,6 +95,8 @@ for n = 1:7
     @test fftshift(fftfreq(n)) ≈ fftshift([fftfreq(n);])
 end
 
+@test isapprox(meanfreq(sin.(2*pi*10*(0:1e-3:10*pi)),1e3),10.0,rtol=0.1)
```

Comment:
Some spaces for readability would be appreciated. This could also be
```julia
@test meanfreq(sin.(2*pi*10*(0:1e-3:10*pi)), 1e3) ≈ 10.0 rtol=0.1
```
But that might be a matter of taste.
Also `rtol` could be lowered to `1e-3`.